dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04310CreateWorkflowRoles.java

```diff
@@ -0,0 +1,178 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.business.WrapInTransaction;
+import com.dotmarketing.business.RoleAPI;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.util.Logger;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Add the Workflow Roles
+ */
+public class Task04310CreateWorkflowRoles extends AbstractJDBCStartupTask {
+
+    private static final String getSystemRole =
+            "select id, role_name, description, role_key, db_fqn, parent, edit_permissions, edit_users, edit_layouts, "
+                    +
+                    "locked, system from cms_role where role_key = 'System' and id = parent";
+
+    private static final String selectSystemRoles =
+            "select id, role_name, description, role_key, db_fqn, parent, edit_permissions, edit_users, edit_layouts, "
+                    +
+                    "locked, system from cms_role where parent = ? and parent <> id";
+
+    private static final String insertRole =
+            "insert into cms_role (id, role_name, description, role_key, db_fqn, parent, edit_permissions, "
+                    +
+                    "edit_users, edit_layouts, locked, system) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
+
+    private static final String AnyoneWhoCanReadRole = "Anyone who can View Content";
+    private static final String AnyoneWhoCanReadRoleId = "0f995057-5c35-4ae7-998a-774dda146c63";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)